uniflow-android-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt

```diff
@@ -15,14 +15,23 @@ class TestObserver<T> : Observer<T> {
 }
 
 data class TestViewObserver(private val states: TestObserver<UIState>, private val events: TestObserver<Event<UIEvent>>) {
-    fun hasState(state: UIState): Boolean = states.elements.lastOrNull() == state
-    fun hasState(state: UIState, index: Int): Boolean = states.elements[index] == state
-    fun lastState(): UIState? = states.elements[states.elements.lastIndex]
-    fun statesCount() = states.elements.count()
-    fun hasEvent(event: UIEvent) = events.elements.lastOrNull() == event
-    fun hasEvent(event: UIEvent, index: Int) = events.elements[index] == event
-    fun eventsCount() = events.elements.count()
-    fun lastEvent(): UIEvent? = events.elements[events.elements.lastIndex].take()
+    fun hasState(state: UIState) = hasState(state, states.elements.lastIndex)
```

Comment:
I'm not sure if it's obvious that this will assert that the _last_ state equals `state`. As a user I might expect that this just asserts that `state` has ever been observed, possibly previously.

Reply:
yes a bit obvious this one